src/ConfigFiles/nativeLang.xml

```diff
@@ -22,7 +22,8 @@
 
 <GUP_NativeLangue name = "English">
 	<PopupMessages>
-		<MSGID_NOUPDATE>No update is available.\r\rThe auto-update may not have been activated yet.\rGo to the official website to check for a new version?</MSGID_NOUPDATE>
+		<MSGID_NOUPDATE1>No update is available.</MSGID_NOUPDATE1>
+		<MSGID_NOUPDATE2>The auto-update may not have been activated yet.</MSGID_NOUPDATE2>

```

Comment:
Sorry, I didn't see (think of) it.
But these 2 lines could be one line:
`<MSGID_NOUPDATE>No update is available.\rThe auto-update may not have been activated yet.</MSGID_NOUPDATE>`
So why bother to separate this line?

Reply:
My logic was because `MSGID_NOUPDATE` is already translated, so I just added a new string.

Also, I had a problem when \r had no effect and was displayed as is.
Just figured out there is a wrap option that should help with that:
https://docs.microsoft.com/en-us/windows/win32/controls/about-static-controls#text-static-control

Ok, I can update it later. Then I'll have to rework the dialog resource a bit.

Reply:
So I tried different styles for this static text control.
For some reason, it keeps showing "\r" as is when I load a string from XML and set it through SetDlgItemTextA.
![image](https://user-images.githubusercontent.com/9664141/106351835-5e0fe700-62e7-11eb-8f96-5ea27e52913e.png)
![image](https://user-images.githubusercontent.com/9664141/106351596-002ecf80-62e6-11eb-8761-f3cbe57ac96e.png)

However, if I don't set the string from code, it looks fine:
![image](https://user-images.githubusercontent.com/9664141/106351719-729faf80-62e6-11eb-90e1-f48c5f87bd8a.png)
![image](https://user-images.githubusercontent.com/9664141/106351738-94009b80-62e6-11eb-9672-5caaf0e511e3.png)

Also, it seems multiline text can't be centered.

So I'd just keep 2 strings. Is it ok?

Reply:
I figured out, its XML loader that adds escaping to "\r":
![image](https://user-images.githubusercontent.com/9664141/106352046-c7dcc080-62e8-11eb-8992-3f933d0331a4.png)
Also, I didn't find any "\r" in `english.xml` in Notepad++.

As I said before, let's keep 2 strings for now.

Reply:
I prefer to have only 1 translation entry, and I don't see the point to keep 2 strings.
Just use the normal Carrier Return in xml as following, it should work:
```
<MSGID_NOUPDATE>No update is available.


The auto-update may not have been activated yet.</MSGID_NOUPDATE>
```